packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/CameraFeatureFactoryImpl.java

```diff
@@ -27,49 +27,56 @@
  * android.hardware.camera2.CaptureRequest}.
  */
 public class CameraFeatureFactoryImpl implements CameraFeatureFactory {
-
+  @NonNull
```

Comment:
Interesting, it doesn't matter where this goes? I would have thought it would need be next to the type (`@Override public @NonNull AutoFocusFeature`).

Reply:
I've didn't even know you could put it after the `public` modifier. But, yea this worked for me. I assumed the same pattern as Dart.